static/js/sections.js

```diff
@@ -113,51 +113,67 @@ $(document).ready(function () {
                     return str;
                 }
             },
-
             {
                 targets: 5,
                 data: null,
+                render: function ( data, type, row, meta ){
+                    let str = "<select id=\""+data['paw']+"-c2\">";
+                    if ( data['c2'] == 'HTTP' ){
+                        str += "<option value=\"HTTP\" selected=\"selected\">HTTP</option>\n" +
+                               "<option value=\"GIST\">GIST</option>\n";
+                    } else {
+                        str += "<option value=\"HTTP\">HTTP</option>\n" +
+                               "<option value=\"GIST\" selected=\"selected\">GIST</option>";
```

Comment:
could set two options and then select via js depending on data['c2'], this would shorten the code block

Reply:
How would you go about selecting the value base on data['c2'] in the scope of the render function using js?

Reply:
it'd look something like 
`$("#select-id").val(data['c2']).select();`

Reply:
Would that need to be called outside of the function scope?